test/plug/ssl_test.exs

```diff
@@ -3,109 +3,185 @@ defmodule Plug.SSLTest do
   import Plug.Test
   import Plug.Conn
 
+  setup_all do
+    tmp_dir = System.tmp_dir!()
```

Comment:
Are you sure these changes are necessary? `configure` doesn't really call `ssl` to validate the certificates, so we don't really need OpenSSL installed. Removing this requirement will make sure that the tests run in more scenarios. So perhaps we should simplify the overall changes to this file to simply update the checks for ciphers and similar?